src/components/Blocks.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+const gameBlocks = require('./../data/gameBlocks.json');
+
+function Blocks() {
+  function blockCreator(element, index) {
```

Comment:
keep this function at the top level. don't nest function inside another one if it doesn't need any variable access from outer environment.